modules/kafka/src/test/java/org/testcontainers/containers/KafkaContainerTest.java

```diff
@@ -29,8 +29,8 @@
     private static final DockerImageName ZOOKEEPER_TEST_IMAGE = DockerImageName.parse("confluentinc/cp-zookeeper:4.0.0");
 
     // junitRule {
-    @Rule
-    public KafkaContainer kafka = new KafkaContainer();
+    @ClassRule
+    public static KafkaContainer kafka = new KafkaContainer(DockerImageName.parse("confluentinc/cp-kafka:5.2.1"));
```

Comment:
heads up: making it static would create false positives since `#testKafkaFunctionality` uses the same topic name / message that it asserts.
Let's change the topic name in `#testKafkaFunctionality` to something with random in it

Reply:
Fixed